app/notification.go

```diff
@@ -468,8 +474,9 @@ func (a *App) sendNotificationEmail(post *model.Post, user *model.User, channel
 /**
  * Computes the subject line for direct notification email messages
  */
-func getDirectMessageNotificationEmailSubject(post *model.Post, translateFunc i18n.TranslateFunc, siteName string, senderName string) string {
+func getDirectMessageNotificationEmailSubject(user *model.User, post *model.Post, translateFunc i18n.TranslateFunc, siteName string, senderName string, useMilitaryTime bool) string {
 	t := getFormattedPostTime(post, translateFunc)
+	t = getLocalTime(user, t, useMilitaryTime, translateFunc)
```

Comment:
I can make that change. 

I would still keep the `getLocalTime` function and just make a call inside `getFormattedPostTime`.  That way it's only making one call instead of 2 calls back to back.

Reply:
Taking a look at `getFormattedPostTime ` it's not that big of a function that it can be repurposed to use timezone preference and militaryTime if they exist